Sources/GithubAPI.swift

```diff
@@ -193,23 +193,49 @@ public class GithubAPI {
     return pathNames
   }
 
+
+  /// Get the project's column name by providing the column ID.
+  ///
+  /// - Parameter columnID: the column ID number.
+  /// - Returns: the name of the column.
```

Comment:
Minor thing, but the MDC repo uses /** */ notation for api docs, so it may be worth considering aligning with that so that we don’t have two standards for the team.

Reply:
opened https://github.com/material-foundation/material-automation/issues/33

Reply:
Normally I also vote in favor of consistency, but Xcode auto-generates comments in Swift files using ///.
To demonstrate press option-command-/ on the line above a function.

Reply:
![comments](https://user-images.githubusercontent.com/45670/40924679-e5e23a70-67e5-11e8-96dd-c868e873559a.gif)

Xcode may be extracting the contextual comment format?

Reply:
Or does it generate different comments for Swift vs Objective-C? >.<

Reply:
Oh noooooo Xcode whyyyyy

Reply:
Am le sad :(